apps/api/src/app/portfolio/portfolio-calculator.ts

```diff
@@ -134,6 +148,14 @@ export class PortfolioCalculator {
   }
 
   public async getCurrentPositions(start: Date): Promise<CurrentPositions> {
+    if (true) {
```

Comment:
In the calculator you do not need conditionals (with the factory described above)